docker-compose.yml

```diff
@@ -1,31 +1,35 @@
 version: '3.1'
 
 services:
-
   server:
     image: deviavir/zenbot:unstable
+    # build: .
     volumes:
-      - ./conf.js:/app/conf.js
-      - ./extensions:/app/extensions
-      - ./simulations:/app/simulations
+      - '.:/app'
     links:
       - mongodb
-    command: [ "trade", "--paper" ]
+    working_dir: /app
+    entrypoint: sh -c
+    command: ["./startup.sh"]
```

Comment:
Why not `trade --paper`?

Reply:
If trade --paper fails, the container won't start. Also, sometimes I don't want it running the trade, so this gives me the ability to trigger it myself or not.